premake/wxwidgets.lua

```diff
@@ -380,6 +387,10 @@ function init_filters()
     system "Windows"
     architecture "x64"
 
+  filter { "platforms:MacOsx" }
+    system "macosx"
+    architecture "x64"
```

Comment:
When adding MacOS support it should not be restricted to `x64`. Modern MacOS computers would require `ARM64` support.